packages/ssl/meta.yaml

```diff
@@ -8,11 +8,14 @@ build:
   script: |
     mkdir dist
     export DISTDIR=$(pwd)/dist
+    export PKG_CONFIG_PATH=${WASM_PKG_CONFIG_PATH}:${PKG_CONFIG_PATH}
```

Comment:
We set this in a large number of packages. Can we set `PKG_CONFIG_PATH` in `buildpkg`?

Reply:
Yeah, I'll try that. Let's see if it doesn't conflict with other packages.